core/src/com/mygdx/game/client/model/ActiveEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.mygdx.game.client.model;
+
+import com.badlogic.ashley.core.Entity;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.logging.Level;
+
+@Singleton
+@Data
+@Log
+public class ActiveEntity {
```

Comment:
Z jednej strony może być, bo dałoby się to używać do UI w postaci scene2d, do wyświetlania itd, ale z drugiej zdaje się być możliwe do zrobienia przez ECS.

Reply:
I nie podoba mi się jakoś to "ActiveEntityType", bo to tak jakby reimplementowanie systemu typów, który trochę zastąpiliśmy ECSem.